tests/data_generator/example_test_data_config.yaml

```diff
@@ -0,0 +1,46 @@
+# list of "global" concepts:
+concepts:
+- concept: HARE-group
+  id: 2000007027
+  concept_name: 'HARE'
+  value_type: concept
+- concept: HARE-group
+  id: 2000007027
```

Comment:
This is a bit confusing to me, but i guess this isn't simply saying what concepts are in the DB, it's also setting how concepts are added as observations?

Reply:
concepts only generate observations if:
- they have a "persons_ids" list (see next yaml example file)
- or if they are nested within a cohort (and thus apply to the persons of that cohort)

Reply:
In this case, these concepts only generate `concept` records.